server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/ChangelogActionMockTest.java

```diff
@@ -45,35 +44,35 @@
 import static org.mockito.Mockito.when;
 import static org.sonar.api.utils.DateUtils.parseDate;
 import static org.sonar.server.qualityprofile.QProfileTesting.XOO_P1_KEY;
-import static org.sonar.server.qualityprofile.ws.ChangelogAction.PARAM_SINCE;
-import static org.sonar.server.qualityprofile.ws.ChangelogAction.PARAM_TO;
-import static org.sonarqube.ws.client.qualityprofile.QualityProfileWsParameters.PARAM_PROFILE_KEY;
+import static org.sonarqube.ws.client.qualityprofile.QualityProfileWsParameters.PARAM_PROFILE;
+import static org.sonarqube.ws.client.qualityprofile.QualityProfileWsParameters.PARAM_SINCE;
+import static org.sonarqube.ws.client.qualityprofile.QualityProfileWsParameters.PARAM_TO;
 
 public class ChangelogActionMockTest {
```

Comment:
we could but there's a competing `ChangelogDatabaseActionTest`… I know, very ugly. I have let is as is for the time being